testsuite/README

```diff
@@ -43,10 +43,10 @@ For questions and discussions for code development, join the developer list http
 
 .. badges
 
-.. |build| image:: https://travis-ci.com/MDAnalysis/mdanalysis.svg?branch=develop
-   :alt: Build Status
-   :target: https://travis-ci.com/MDAnalysis/mdanalysis
-
+.. |build| image:: https://github.com/MDAnalysis/mdanalysis/actions/workflows/gh-ci.yaml/badge.svg
+   :alt: Github Actions Build Status
+   :target: https://github.com/MDAnalysis/mdanalysis/actions/workflows/gh-ci.yaml
+   
```

Comment:
```suggestion

```
Excess white spaces don't do anything but it's nice not to have them.

Reply:
I have added myself in both AUTHORS and CHANGELOG list. Is it correct?

Reply:
This white space issue still need addressing.